srcfile.go

```diff
@@ -37,6 +40,19 @@ func (f *SrcFile) Position(offset int) Position {
 	if line >= 0 {
 		lineStart = f.lineOffsets[line]
 	}
+
+	row := line + 2
+	col := offset - lineStart + 1
+
+	if f.sourceMap != nil {
+		if _, _, row, col, ok := f.sourceMap.Source(row, col); ok {
```

Comment:
What about file name and function name?

Reply:
Not sure where that would be exchanged with the sourcemap values. For me (since I load all the files myself) the values are already correct (the typescript compiler does not change method names).

Reply:
It will require a bit of refactoring... I'm going to merge this and get to it a bit later.

Reply:
👍 sure :)